tests/Filesystem/ReplicateTest.php

```diff
@@ -21,40 +21,40 @@ class ReplicateTest extends TestCase
 {
     public function testReplicate(): void
```

Comment:
Can you add a test that checks the BC layer

Reply:
This file is not related to the current deprecation, this change is only about removing some master and slave words